src/core/tailer/tailer.go

```diff
@@ -95,6 +101,28 @@ func NewPatternTailer(file string, patterns map[string]string) (*PatternTailer,
 	return &PatternTailer{t, gc}, nil
 }
 
+func NewLTSVTailer(file string) (*LTSVTailer, error) {
+	ltsvParseString := func(line string) map[string]string {
+		columns := strings.Split(line, "\t")
+		lvs := make(map[string]string)
```

Comment:
Not sure what lvs and lv mean. Could you give the variables more meaningful names?

Reply:
@dhurley Also Fix those name.